README.md

```diff
@@ -25,10 +25,15 @@ for Google Cloud Platform :
 
 * Instantiate an image with the resulting AMI
   * Make sure it has a public IP address
-  * Make sure it is in a security group with the following inbound rules:
+  * Make sure it is in a security group with at least the following inbound rules:
     * UDP 500
     * UDP 4500
     * TCP 22 to your IP (for SSH management)
+  * Make sure it is in a security group with at least the following outbound rules:
+    * UDP 500
+    * UDP 4500
```

Comment:
Seems right but did you test these?

Reply:
I had checked what ports the two sides are listening on, turns out we need ALL UDP